.github/workflows/ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         MOCK_MAKER: ${{ matrix.entry.mock-maker }}
         MEMBER_ACCESSOR: ${{ matrix.entry.member-accessor }}
 
-    - name: 7. Upload coverage report
+    - name: 10. Upload coverage report
```

Comment:
maths :D

Reply:
:D yeah, adding and removing steps, will fix that 😄